esmvalcore/_recipe/recipe.py

```diff
@@ -250,25 +250,45 @@ def _add_to_download_list(dataset):
 
 
 def _schedule_for_download(datasets):
-    """Schedule files for download and show the list of files in the log."""
+    """Schedule files for download."""
     for dataset in datasets:
         _add_to_download_list(dataset)
         for supplementary_ds in dataset.supplementaries:
             _add_to_download_list(supplementary_ds)
 
-        files = list(dataset.files)
-        for supplementary_ds in dataset.supplementaries:
-            files.extend(supplementary_ds.files)
+
+def _log_all_files(datasets: Iterable[Dataset]) -> None:
```

Comment:
Maybe rename to `_log_input_files`?

Reply:
Done in https://github.com/ESMValGroup/ESMValCore/pull/2303/commits/69bfde561f106b905e1457196e822bc6b561a2a7